src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/AddDraftApprenticeshipViewModelMapper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.Apprenticeships.Api.Types;
+using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.ProviderCommitments.Queries.GetTrainingCourses;
+using SFA.DAS.ProviderCommitments.Web.Models;
+using SFA.DAS.CommitmentsV2.Shared.Models;
+
+namespace SFA.DAS.ProviderCommitments.Web.Mappers.Cohort
+{
+    public class AddDraftApprenticeshipViewModelMapper : IMapper<CreateCohortWithDraftApprenticeshipRequest, AddDraftApprenticeshipViewModel>
+    {
+        private readonly ICommitmentsApiClient _commitmentsApiClient;
+        private readonly IMediator _mediator;
+
+
```

Comment:
Presumably this space is for ventilation?

Reply:
I do try to find these blank lines, before I submit a PR. I'm convinced someone else is adding them